docs/start_nominating.md

```diff
@@ -107,7 +107,7 @@ To nominate the selected validators, click _Nominate_ and sign the transaction.
 
 ## 03 Check the status of your nominations
 
-After completing the nominating process, your nominations will be inactive for the remainder of the current era. Once the following era begins, your nominations will become active, provided that the validator nodes you have nominated are included in the active set of validators. If a given validator remains in the waiting queue, your respective nomination will remain inactive and you will not earn any rewards for this stake.
+After completing the nominating process, your nominations will be inactive for the remainder of the current era. Once the following era begins, your nominations will become active, provided that at least one of the validator nodes you have nominated is included in the active set of validators and is not overcrowded leaving you out. If all of your validators remain in the waiting queue, your respective nominations will remain inactive too and you will not earn any rewards for this era.
```

Comment:
After completing the nominating process, your nominations will be inactive for the remainder of the current era. Once the following era begins, your nominations will become active, provided that at least one of your nominated validator nodes is included in the active set of validators, and that your nomination is not disregarded because the validator was oversubscribed. If all of your validators remain in the waiting queue, your respective nominations will remain inactive too and you will not earn any rewards for the duration of the era.